distributed/scheduler.py

```diff
@@ -1298,6 +1298,8 @@ def __init__(
         self.log = deque(
             maxlen=dask.config.get("distributed.scheduler.transition-log-length")
         )
+        self.events = defaultdict(lambda: deque(maxlen=100000))
+        self.event_counts = defaultdict(lambda: 0)
```

Comment:
If you wanted to be ultra-concise

```suggestion
        self.event_counts = defaultdict(int)
```

Reply:
+1